libs/entry-components/styles/modules/components/inputs/_generator.scss

```diff
@@ -11,5 +12,17 @@
 				border-radius: 4px;
 			}
 		}
+
```

Comment:
What exactly you're trying to solve with this? Background color on focus? We probably want to skip that, since grayish color is everywhere on the focus and it's not the way.